include/RAJA/util/PluginOptions.hpp

```diff
@@ -0,0 +1,31 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_Plugin_Options_HPP
+#define RAJA_Plugin_Options_HPP
+
+namespace RAJA {
+namespace util {
+
+#include <string>
+
+struct PluginOptions
+{
+    PluginOptions(const std::string newstr) : str(newstr) {};
```

Comment:
parameter should be passed by const reference

Reply:
👍